app/controllers/application_controller.rb

```diff
@@ -69,114 +69,106 @@ def are_allies(userid1, userid2)
     userid2 = User.find(userid2)
     is_allies_userid1 = userid1.allies_by_status(:accepted).include?(userid2)
     is_allies_userid2 = userid2.allies_by_status(:accepted).include?(userid1)
-    return is_allies_userid1 && is_allies_userid2
+    is_allies_userid1 && is_allies_userid2
   end
 
   def is_viewer(viewers)
-    if (viewers.include? current_user.id)
-      return true
-    end
-
-    return false
+    viewers.include? current_user.id
   end
 
   def get_uid(userid)
-    uid = User.where(id: userid).first.uid
-    return uid
+    User.where(id: userid).first.uid
   end
 
   def most_focus(data_type, profile)
-    data = Array.new
+    data = []
 
-    if profile.blank?
-      userid = current_user.id
-    else
-      userid = profile
-    end
+    userid =
```

Comment:
Ah ok! Do you mind updating it to a ternary operator then? :) I prefer to minimize lines when possible (and when the conditional isn't complicated)